projects/1.16/assets/agricraft/agricraft/lang/zh_cn.json

```
@@ -0,0 +1,847 @@
+  "agricraft.plant.mysticalagriculture.zombie": "僵尸种子",
+  "agricraft.plant.mysticalagriculture.skeleton": "骷髅种子",
+  "agricraft.plant.mysticalagriculture.creeper": "苦力怕种子",
+  "agricraft.plant.mysticalagriculture.spider": "蜘蛛种子",
+  "agricraft.plant.mysticalagriculture.rabbit": "兔子种子",
+
+  "agricraft.plant.mysticalagriculture.gold": "金种子",
+  "agricraft.plant.mysticalagriculture.lapis_lazuli": "青金石种子",
+  "agricraft.plant.mysticalagriculture.end": "末地种子",
+  "agricraft.plant.mysticalagriculture.experience": "经验种子",
+  "agricraft.plant.mysticalagriculture.blaze": "烈焰人种子",
+  "agricraft.plant.mysticalagriculture.ghast": "恶魂种子",
+  "agricraft.plant.mysticalagriculture.enderman": "末影人种子",
+
+  "agricraft.plant.mysticalagriculture.diamond": "钻石种子",
+  "agricraft.plant.mysticalagriculture.emerald": "绿宝石种子",
+  "agricraft.plant.mysticalagriculture.netherite": "下界合金种子",
+  "agricraft.plant.mysticalagriculture.wither_skeleton": "凋灵骷髅种子",
+
+  "agricraft.plant.mysticalagriculture.rubber": "橡胶种子",
+  "agricraft.plant.mysticalagriculture.silicon": "硅种子",
+  "agricraft.plant.mysticalagriculture.sulfur": "硫种子",
+  "agricraft.plant.mysticalagriculture.aluminum": "铝种子",
+  "agricraft.plant.mysticalagriculture.copper": "铜种子",
+  "agricraft.plant.mysticalagriculture.saltpeter": "硝石种子",
+  "agricraft.plant.mysticalagriculture.tin": "锡种子",
+  "agricraft.plant.mysticalagriculture.bronze": "青铜种子",
+  "agricraft.plant.mysticalagriculture.zinc": "锌种子",
+  "agricraft.plant.mysticalagriculture.brass": "黄铜种子",
+  "agricraft.plant.mysticalagriculture.silver": "银种子",
+  "agricraft.plant.mysticalagriculture.lead": "铅种子",
+  "agricraft.plant.mysticalagriculture.graphite": "石墨种子",
+  "agricraft.plant.mysticalagriculture.steel": "钢种子",
+  "agricraft.plant.mysticalagriculture.nickel": "镍种子",
+  "agricraft.plant.mysticalagriculture.constantan": "康铜种子",
+  "agricraft.plant.mysticalagriculture.electrum": "琥珀金种子",
+  "agricraft.plant.mysticalagriculture.invar": "因瓦种子",
+  "agricraft.plant.mysticalagriculture.mithril": "秘银种子",
+  "agricraft.plant.mysticalagriculture.tungsten": "钨种子",
+  "agricraft.plant.mysticalagriculture.titanium": "钛种子",
+  "agricraft.plant.mysticalagriculture.uranium": "铀种子",
+  "agricraft.plant.mysticalagriculture.chrome": "铬种子",
+  "agricraft.plant.mysticalagriculture.platinum": "铂种子",
+  "agricraft.plant.mysticalagriculture.iridium": "铱种子",
+
+  "agricraft.plant.mysticalagriculture.apatite": "磷灰石种子",
+  "agricraft.plant.mysticalagriculture.ruby": "红宝石种子",
+  "agricraft.plant.mysticalagriculture.sapphire": "蓝宝石种子",
+
+  "agricraft.plant.mysticalagriculture.blizz": "暴雪种子",
+  "agricraft.plant.mysticalagriculture.blitz": "狂风种子",
+  "agricraft.plant.mysticalagriculture.basalz": "岩石种子",
+  "agricraft.plant.mysticalagriculture.signalum": "信素种子",
+  "agricraft.plant.mysticalagriculture.lumium": "流明种子",
+  "agricraft.plant.mysticalagriculture.enderium": "末影种子",
+
+  "agricraft.plant.mysticalagriculture.hop_graphite": "HOP石墨种子",
```

```suggestion
  "agricraft.plant.mysticalagriculture.hop_graphite": "高定向热解石墨种子",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/2038b5fdd9bf8fa831d950cce1a765fc18f9b008/projects/1.16/assets/immersive-engineering/immersiveengineering/lang/zh_cn.json#L1018-L1019